src/main/scala/no/ndla/draftapi/integration/ConceptApiClient.scala

```diff
@@ -33,8 +33,8 @@ trait ConceptApiClient {
     private val conceptTimeout = 1000 * 10 // 10 seconds
 
     def publishConceptsIfToPublishing(ids: Seq[Long]): Seq[Try[DraftConcept]] = {
-      val statusNotToPublish = "PUBLISHED"
-      val shouldPublish = (c: DraftConcept) => c.status.current != statusNotToPublish
+      val statusToPublish = "QUALITY_ASSURED"
```

Comment:
Kanskje dette skal være ei liste med statuser? For publisering av publiserte forklaringer gir ikkje meining.

Reply:
Ah du har snudd sjekken. Gløm denne kommentaren.

Reply:
Linje 44 må vel endres til å gjenspeile korrekt oppførsel.